book/src/developer-guides.md

```diff
@@ -0,0 +1,6 @@
+# **💻** Developer Guides
+
+
+Here are the articles in this section:
+
+[Composable SDK](./developer-guides/composable-sdk.md)
```

Comment:
Though this is correct with the live version, in this version there is another chapter called "Pallet Overviews".

Reply:
These are a recent addition from me that I had finished after the migration began. Part of a larger goal making our technical documentation more approachable. Would we prefer that this be moved to its own PR?

Reply:
Completely fine with adding this to the PR. Sorry been a bit unclear. Just wanted to have the link added to this index doc.

I don't know how to suggest changes on GitHub, so this is basically what i intended to have here:
```
- [Composable SDK](./developer-guides/composable-sdk.md)
- [Pallet Overviews](./developer-guides/pallet-overviews.md)
```

Reply:
I like this suggestion, will add it @Dom-Roth

Reply:
Fixed in 2fb2435b24b17e33f77f2f9e7f5782bdb694f442